src/usr/local/emhttp/plugins/tailscale/include/Tailscale/System.php

```
@@ -66,6 +66,56 @@
+                     ? intval($ident_config['PORT']) : 80;
+        $httpsPort = -1;
+
+        if (($ident_config['USE_SSL'] ?? "no") != "no") {
+            $httpsPort = isset($ident_config['PORTSSL']) && is_scalar($ident_config['PORTSSL'])
+                         ? intval($ident_config['PORTSSL']) : 443;
+        }
+
+        $localAPI    = new LocalAPI();
+        // …
+    }
+    $localAPI    = new LocalAPI();
```

PHPStan (2.0.3)

Syntax error, unexpected T_VARIABLE on line 85


(phpstan.parse)